### PR TITLE
reformat all c/cpp/h files with astyle config

### PR DESCRIPTION
**Pull request by slabasan**

### CODE REVIEW

File: src/variorum/Intel/Haswell_3F.c [hunk 1]
```diff
@@ -144,52 +149,95 @@ int fm_06_3f_get_features(void)
     printf("Running %s\n", __FUNCTION__);
 #endif
```
Comment (tpatki) [inline src/variorum/Intel/Haswell_3F.c]: Is this the right indentation for `#endif`? Didn't take a look at the raw file (too late here).

Comment (tpatki) [reply src/variorum/Intel/Haswell_3F.c]: Not sure if it is rendering incorrectly. Other `ifdefs` and `endifs` look different on my review screen..

File: src/variorum/Intel/clocks_features.c [hunk 2]
```diff
@@ -112,54 +116,62 @@ void dump_clocks_data(FILE *writedest, off_t msr_aperf, off_t msr_mperf, off_t m
...
         {
-            fprintf(writedest, "_CLOCKS_DATA hostname socket aperf mperf tsc curr_freq_mhz avg_freq_mhz\n");
+            fprintf(writedest,
+                    "_CLOCKS_DATA hostname socket aperf mperf tsc curr_freq_mhz avg_freq_mhz\n");
         }
         else if (control_domains == CORE)
         {
-            fprintf(writedest, "_CLOCKS_DATA hostname socket core thread_phy thread_log aperf mperf tsc curr_freq_mhz avg_freq_mhz\n");
+            fprintf(writedest,
+                    "_CLOCKS_DATA hostname socket core thread_phy thread_log aperf mperf tsc curr_freq_mhz avg_freq_mhz\n");
```
Comment (tpatki) [inline src/variorum/Intel/clocks_features.c]: Looks longer than 80 chars...

File: src/variorum/Intel/clocks_features.c [hunk 3]
```diff
@@ -177,39 +189,45 @@ void print_clocks_data(FILE *writedest, off_t msr_aperf, off_t msr_mperf, off_t
...
                 {
-                    idx = (k * nsockets * (ncores/nsockets)) + (i * (ncores/nsockets)) + j;
-                    fprintf(writedest, "_CLOCKS_DATA Hostname: %s Socket: %d APERF: %lu MPERF: %lu TSC: %lu Curr_Freq_MHz: %lu Avg_Freq_MHz: %f\n",
-                            hostname, i, *cd->aperf[idx], *cd->mperf[idx], *cd->tsc[idx], MASK_VAL(*pd->perf_status[i], 15, 8) * 100, max_non_turbo_ratio*((*cd->aperf[idx])/(double)(*cd->mperf[idx])));
+                    for (k = 0; k < nthreads / ncores; k += nthreads / ncores)
+                    {
+                        idx = (k * nsockets * (ncores / nsockets)) + (i * (ncores / nsockets)) + j;
+                        fprintf(writedest,
+                                "_CLOCKS_DATA Hostname: %s Socket: %d APERF: %lu MPERF: %lu TSC: %lu Curr_Freq_MHz: %lu Avg_Freq_MHz: %f\n",
+                                hostname, i, *cd->aperf[idx], *cd->mperf[idx], *cd->tsc[idx],
```
Comment (tpatki) [inline src/variorum/Intel/clocks_features.c]: Longer than 80 characters?

File: src/variorum/Intel/clocks_features.c [hunk 4]
```diff
@@ -177,39 +189,45 @@ void print_clocks_data(FILE *writedest, off_t msr_aperf, off_t msr_mperf, off_t
...
+                for (j = 0; j < ncores / nsockets; j++)
                 {
-                    idx = (k * nsockets * (ncores/nsockets)) + (i * (ncores/nsockets)) + j;
-                    fprintf(writedest, "_CLOCKS_DATA Hostname: %s Socket: %d Core: %d Thread_Phy: %d Thread_Log: %d APERF: %lu MPERF: %lu TSC: %lu Curr_Freq_Mhz: %lu Avg_Freq_Mhz: %f\n",
-                            hostname, i, j, k, idx, *cd->aperf[idx], *cd->mperf[idx], *cd->tsc[idx], MASK_VAL(*pd->perf_status[i], 15, 8) * 100, max_non_turbo_ratio*((*cd->aperf[idx])/(double)(*cd->mperf[idx])));
+                    for (k = 0; k < nthreads / ncores; k++)
+                    {
+                        idx = (k * nsockets * (ncores / nsockets)) + (i * (ncores / nsockets)) + j;
+                        fprintf(writedest,
+                                "_CLOCKS_DATA Hostname: %s Socket: %d Core: %d Thread_Phy: %d Thread_Log: %d APERF: %lu MPERF: %lu TSC: %lu Curr_Freq_Mhz: %lu Avg_Freq_Mhz: %f\n",
```
Comment (tpatki) [inline src/variorum/Intel/clocks_features.c]: Longer than 80 characters?

File: src/variorum/Intel/clocks_features.c [hunk 5]
```diff
@@ -289,34 +308,34 @@ void set_p_state(int cpu_freq_mhz, enum ctl_domains_e domain, off_t msr_perf_sta
...
+        case CORE:
+            printf("Set frequencies per core\n");
+            for (i = 0; i < nthreads; i++)
+            {
+                *pd->perf_ctl[i] = cpu_freq_mhz / 100 * 256;
+            }
 #if VARIORUM_DEBUG
-        printf("PERF_CTL raw decimal %" PRIu64 "\n", *pd->perf_ctl[9]);
+            printf("PERF_CTL raw decimal %" PRIu64 "\n", *pd->perf_ctl[9]);
 #endif
```
Comment (tpatki) [inline src/variorum/Intel/clocks_features.c]: What is the correct indentation for code within an `#if` and `#endif` block? I saw a few places where there was no indentation, but I see the code being indented here?

File: src/variorum/Intel/counters_features.c [hunk 6]
```diff
@@ -376,35 +422,46 @@ void print_perfmon_counter_data(FILE *writedest, off_t *msrs_perfevtsel_ctrs, of
         switch (avail)
         {
             case 8:
-                fprintf(writedest, "_PERFORMANCE_MONITORING_COUNTERS Host: %s Thread: %d PMC0: %lu PMC1: %lu PMC2: %lu PMC3: %lu PMC4: %lu PMC5: %lu PMC6: %lu PMC7: %lu\n",
-                        hostname, i, *p->pmc0[i], *p->pmc1[i], *p->pmc2[i], *p->pmc3[i], *p->pmc4[i], *p->pmc5[i], *p->pmc6[i], *p->pmc7[i]);
+                fprintf(writedest,
+                        "_PERFORMANCE_MONITORING_COUNTERS Host: %s Thread: %d PMC0: %lu PMC1: %lu PMC2: %lu PMC3: %lu PMC4: %lu PMC5: %lu PMC6: %lu PMC7: %lu\n",
```
Comment (tpatki) [inline src/variorum/Intel/counters_features.c]: More than 80 chars?

File: src/variorum/Intel/counters_features.c [hunk 7]
```diff
@@ -376,35 +422,46 @@ void print_perfmon_counter_data(FILE *writedest, off_t *msrs_perfevtsel_ctrs, of
...
+                        hostname, i, *p->pmc0[i], *p->pmc1[i], *p->pmc2[i], *p->pmc3[i], *p->pmc4[i],
+                        *p->pmc5[i], *p->pmc6[i], *p->pmc7[i]);
                 break;
             case 7:
-                fprintf(writedest, "_PERFORMANCE_MONITORING_COUNTERS Host: %s Thread: %d PMC0: %lu PMC1: %lu PMC2: %lu PMC3: %lu PMC4: %lu PMC5: %lu PMC6: %lu\n",
-                        hostname, i, *p->pmc0[i], *p->pmc1[i], *p->pmc2[i], *p->pmc3[i], *p->pmc4[i], *p->pmc5[i], *p->pmc6[i]);
+                fprintf(writedest,
+                        "_PERFORMANCE_MONITORING_COUNTERS Host: %s Thread: %d PMC0: %lu PMC1: %lu PMC2: %lu PMC3: %lu PMC4: %lu PMC5: %lu PMC6: %lu\n",
+                        hostname, i, *p->pmc0[i], *p->pmc1[i], *p->pmc2[i], *p->pmc3[i], *p->pmc4[i],
```
Comment (tpatki) [inline src/variorum/Intel/counters_features.c]: More than 80 chars?

File: src/variorum/Intel/counters_features.c [hunk 8]
```diff
@@ -376,35 +422,46 @@ void print_perfmon_counter_data(FILE *writedest, off_t *msrs_perfevtsel_ctrs, of
...
                         hostname, i, *p->pmc0[i], *p->pmc1[i], *p->pmc2[i], *p->pmc3[i], *p->pmc4[i]);
                 break;
             case 4:
-                fprintf(writedest, "_PERFORMANCE_MONITORING_COUNTERS Host: %s Thread: %d PMC0: %lu PMC1: %lu PMC2: %lu PMC3: %lu\n",
+                fprintf(writedest,
+                        "_PERFORMANCE_MONITORING_COUNTERS Host: %s Thread: %d PMC0: %lu PMC1: %lu PMC2: %lu PMC3: %lu\n",
                         hostname, i, *p->pmc0[i], *p->pmc1[i], *p->pmc2[i], *p->pmc3[i]);
                 break;
             case 3:
-                fprintf(writedest, "_PERFORMANCE_MONITORING_COUNTERS Host: %s Thread: %d PMC0: %lu PMC1: %lu PMC2: %lu\n",
```
Comment (tpatki) [inline src/variorum/Intel/counters_features.c]: Several lines here seem to be longer than 80 chars (require scrolling on my github screen). Is there a better way to test this during review, or should I check out the PR and test with vim?

File: src/variorum/Intel/msr_core.c [hunk 9]
```diff
@@ -87,7 +93,9 @@ static int compatibility_batch(int batchnum, int type)
     struct msr_batch_array *batch = NULL;
     int i;
 
-    fprintf(stderr, "Warning: <variorum> No /dev/cpu/msr_batch, using compatibility batch: compatibility_batch(): %s: %s:%s::%d\n", strerror(errno), getenv("HOSTNAME"), __FILE__, __LINE__);
+    fprintf(stderr,
+            "Warning: <variorum> No /dev/cpu/msr_batch, using compatibility batch: compatibility_batch(): %s: %s:%s::%d\n",
+            strerror(errno), getenv("HOSTNAME"), __FILE__, __LINE__);
```
Comment (tpatki) [inline src/variorum/Intel/msr_core.c]: More than 80?

File: src/variorum/Intel/power_features.c [hunk 10]
```diff
@@ -103,28 +108,34 @@ static int translate(const unsigned socket, uint64_t *bits, double *units, int t
             /* Store the bits in the Intel specified format. */
             *bits = (uint64_t)(timeval_y | (timeval_z << 5));
 #ifdef LIBMSR_DEBUG
-            fprintf(stderr, "%s %s::%d DEBUG: timeval_z is %lx, timeval_y is %lx, units is %lf, bits is %lx, remainder is %lf\n", getenv("HOSTNAME"), __FILE__, __LINE__, timeval_z, timeval_y, *units, *bits, logremainder);
+            fprintf(stderr,
+                    "%s %s::%d DEBUG: timeval_z is %lx, timeval_y is %lx, units is %lf, bits is %lx, remainder is %lf\n",
```
Comment (tpatki) [inline src/variorum/Intel/power_features.c]: More than 80?

File: src/variorum/Intel/power_features.c [hunk 11]
```diff
@@ -339,27 +360,30 @@ int get_rapl_power_unit(struct rapl_units *ru, off_t msr)
...
+    //    uint64_t *tmp = (uint64_t *) libmsr_calloc(sockets, sizeof(uint64_t));
+    //    for (i = 0; i < sockets; i++)
+    //    {
+    //        read_msr_by_coord(i, 0, 0, MSR_RAPL_POWER_UNIT, tmp);
+    //        double energy = (double)(1 << (MASK_VAL(ru[i].msr_rapl_power_unit, 12, 8)));
+    //        double seconds = (double)(1 << (MASK_VAL(ru[i].msr_rapl_power_unit, 19, 16)));
+    //        double power = ((1.0)/((double)(1 << (MASK_VAL(ru[i].msr_rapl_power_unit, 3, 0)))));
+    //        if (energy != ru[i].joules || power != ru[i].watts || seconds != ru[i].seconds)
+    //        {
+    //            variorum_error_handler("Inconsistent rapl power units across packages", VARIORUM_ERROR_RUNTIME, getenv("HOSTNAME"), __FILE__, __FUNCTION__, __LINE__);
```
Comment (tpatki) [inline src/variorum/Intel/power_features.c]: More than 80?

File: src/variorum/Intel/power_features.c [hunk 12]
```diff
@@ -399,7 +426,10 @@ void print_rapl_power_unit(FILE *writedest, off_t msr)
 
     for (socket = 0; socket < nsockets; socket++)
     {
-        fprintf(writedest, "_RAPL_POWER_UNITS Offset: 0x%lx Host: %s Socket: %d Bits: 0x%lx PowerUnit: %fW EnergyUnit: %fJ TimeUnit: %f\n", msr, hostname, socket, ru[socket].msr_rapl_power_unit, ru[socket].watts, 1/ru[socket].joules, 1/ru[socket].seconds);
+        fprintf(writedest,
+                "_RAPL_POWER_UNITS Offset: 0x%lx Host: %s Socket: %d Bits: 0x%lx PowerUnit: %fW EnergyUnit: %fJ TimeUnit: %f\n",
```
Comment (tpatki) [inline src/variorum/Intel/power_features.c]: More than 80?

File: src/variorum/Intel/power_features.c [hunk 13]
```diff
@@ -433,11 +466,15 @@ void print_package_power_info(FILE *writedest, off_t msr, int socket)
 
     if (!get_rapl_power_info(socket, &info, msr))
     {
-        fprintf(writedest, "_PACKAGE_POWER_INFO Offset: 0x%lx Host: %s Socket: %d Bits: 0x%lx MaxPower: %lfW MinPower: %lfW MaxWindow: %lf sec ThermPower: %lfW\n", msr, hostname, socket, info.msr_pkg_power_info, info.pkg_max_power, info.pkg_min_power, info.pkg_max_window, info.pkg_therm_power);
+        fprintf(writedest,
+                "_PACKAGE_POWER_INFO Offset: 0x%lx Host: %s Socket: %d Bits: 0x%lx MaxPower: %lfW MinPower: %lfW MaxWindow: %lf sec ThermPower: %lfW\n",
```
Comment (tpatki) [inline src/variorum/Intel/power_features.c]: More than 80?

File: src/variorum/Nvidia/power_features.c [hunk 14]
```diff
@@ -222,13 +238,15 @@ void dump_gpu_utilization(FILE *writedest, int verbose)
...
+        for (d = device_index * gpus_per_socket;
+             d < (device_index + 1) * gpus_per_socket; ++d)
         {
             nvmlDeviceGetUtilizationRates(m_unit_devices_file_desc[d], &util);
 
             if (verbose)
             {
-                fprintf(writedest, "_GPU_UTILIZATION Host: %s, Socket: %d, Device ID: %d, GPU Utilization (%): SM: %d, Memory: %d\n",
+                fprintf(writedest,
+                        "_GPU_UTILIZATION Host: %s, Socket: %d, Device ID: %d, GPU Utilization (%): SM: %d, Memory: %d\n",
```
Comment (tpatki) [inline src/variorum/Nvidia/power_features.c]: More than 80?

File: src/variorum/Intel/thermal_features.c [hunk 15]
```diff
@@ -299,29 +306,32 @@ int dump_therm_temp_reading(FILE *writedest, off_t msr_therm_stat, off_t msr_pkg
...
+    t_target = (struct msr_temp_target *) malloc(nsockets * sizeof(
+                   struct msr_temp_target));
     get_temp_target(t_target, msr_temp_target);
 
     t_stat = (struct therm_stat *) malloc(nthreads * sizeof(struct therm_stat));
     get_therm_stat(t_stat, msr_therm_stat);
 
-    fprintf(writedest, "_THERMALS socket phys_core log_thread TCC_celsius pkg_reading_celsius pkg_actual_celsius thread_reading_celsius thread_actual_celsius thread_digital_reading_valid\n");
+    fprintf(writedest,
+            "_THERMALS socket phys_core log_thread TCC_celsius pkg_reading_celsius pkg_actual_celsius thread_reading_celsius thread_actual_celsius thread_digital_reading_valid\n");
```
Comment (tpatki) [inline src/variorum/Intel/thermal_features.c]: More than 80?

File: src/variorum/IBM/Power9.c [hunk 16]
```diff
@@ -265,9 +279,11 @@ int p9_set_gpu_power_ratio(int gpu_power_ratio)
 
     usleep(100000);
 
-    fprintf(stdout, "Changed power shifting ratio on both sockets to %d percent.\n", gpu_power_ratio);
+    fprintf(stdout, "Changed power shifting ratio on both sockets to %d percent.\n",
+            gpu_power_ratio);
     fprintf(stdout, "\nNOTE: \n");
-    fprintf(stdout, "  A 100ms delay was added to allow for file operations to propagate out of band.\n"
+    fprintf(stdout,
+            "  A 100ms delay was added to allow for file operations to propagate out of band.\n"
```
[the remark this answers is not in the document]
Comment (slabasan) [reply src/variorum/IBM/Power9.c]: Keeping the space for stdout formatting.

File: src/variorum/IBM/ibm_sensors.c [hunk 17]
```diff
@@ -280,12 +288,13 @@ void print_all_sensors(int chipid, FILE *output, const void *buf)
 
             // Note that we're not capturing timestamp here, the common timestamp printed
             // is the one from the beginning of the loop.
-            fprintf(output, " %lu %lu", (uint64_t)(sample * TO_FP(scale)), (uint64_t)(energy / TO_FP(freq)));
+            fprintf(output, " %lu %lu", (uint64_t)(sample * TO_FP(scale)),
```
[the remark this answers is not in the document]
Comment (slabasan) [reply src/variorum/IBM/ibm_sensors.c]: Keeping the space for stdout formatting.

File: src/variorum/Intel/clocks_features.c (hunk 2 above)
Comment (slabasan) [reply src/variorum/Intel/clocks_features.c]: Astyle counts characters not taking into account the indentation” when enforcing line width. It removes the indentation before calculating the line width. I don't think we can get around this one easily.

File: src/variorum/Intel/clocks_features.c (hunk 5 above)
Comment (slabasan) [reply src/variorum/Intel/clocks_features.c]: This has been indented to accommodate indented switch case statement.

**Pull request merged**